Formula/n/netdata.rb

```diff
@@ -33,72 +31,40 @@ class Netdata < Formula
   depends_on "openssl@3"
   depends_on "pcre2"
   depends_on "protobuf"
-  depends_on "protobuf-c"
+  depends_on "snappy"
 
+  uses_from_macos "curl"
   uses_from_macos "zlib"
 
   on_linux do
+    depends_on "bison" => :build
+    depends_on "flex" => :build
```

Comment:
Not `uses_from_macos`?

Reply:
Not used on macOS since feature that needs bison/flex is Linux-only.